uncontrast_studies/tests/test_journals.py

```diff
@@ -0,0 +1,132 @@
+from rest_framework import status
+
+from contrast_api.choices import StudyTypeChoices, PresentationModeChoices, SignificanceChoices, UnConSampleChoices
+from uncontrast_studies.open_api_parameters import UNCONTRAST_GRAPH_BREAKDOWN_OPTIONS
+from uncontrast_studies.tests.base import UnContrastBaseTestCase
+
+
+class TestJournalsGraphTestCase(UnContrastBaseTestCase):
+    def _setup_world(self):
```

Comment:
why not do this as part of `UnContrastBaseTestCase`? it seems like a lot of repetition

Reply:
It's a dilemma, as there are some variants between them (not a lot) and because it's easier while crafting the more specific test later to "see" the whole "world setup" in the same class . So I'm not sure I know enough to move it to the base class. I plan to do it later in the process